plugins/kubernetes/app/models/kubernetes/job_executor.rb

```diff
@@ -0,0 +1,244 @@
+# executes a deploy and writes log to job output
```

Comment:
either extract static methods to share logic or some other shared object to avoid all this copy-paste ...